### DISCUSSION TITLE
Italian translation is not working...

### DISCUSSION THREAD

**Discussion by marcobra** [pawelsalawa/sqlitestudio#4432] [General]
### Details
Italian translation lang is not working

Steps to reproduce
Change lang set it to  "Italiano" and restart sqlite studio the interface is not translated and italian translation is entirely translated

Operating system
linux

SQLiteStudio version
3.3.3 but italian ready to use translation also not working with previous releases

**Comment by pawelsalawa** [comment]
Where did you get the information that in 3.3.3 the translation is done?
https://github.com/pawelsalawa/sqlitestudio/blob/3.3/SQLiteStudio3/guiSQLiteStudio/translations/guiSQLiteStudio_it.ts
This is the version from 3.3 branch and looking through it, it is 100% NOT translated.

**Comment by pawelsalawa** [comment]
Note, that you can download 3.4.0 nightly builds from https://github.com/pawelsalawa/sqlitestudio/actions

**Comment by marcobra** [comment]
I just translated 100% Italian lang now https://crowdin.com/project/sqlitestudio/it

**Comment by pawelsalawa** [comment]
I've merged the translation Pull Request. It will be included in the upcoming nightly build.
Are you okay with moving this into a discussion? It doesn't seem to be an issue.

**Comment by marcobra** [comment]
yes

Il giorno lun 11 apr 2022 alle ore 23:23 Paweł Salawa <
***@***.***> ha scritto:

> I've merged the translation Pull Request. It will be included in the
> upcoming nightly build.
> Are you okay with moving this into a discussion? It doesn't seem to be an
> issue.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pawelsalawa/sqlitestudio/issues/4430#issuecomment-1095579638>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAMINWZOJTVYOC4PQ6IXCHTVESJ3LANCNFSM5TAVO3PQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Linux Infinite Freedom

I'm writing from this place:
http://www.openstreetmap.org/?lat=44.39945&lon=8.6798&zoom=15&layers=M

**Reply by pawelsalawa** [reply]
Okay. Thank you for the contribution!

**Comment by marcobra** [comment]
i've installed the latest nightly 2022-04-13 dev 3.4.0 but no italian translation is available ;-(

**Reply by pawelsalawa** [reply]
I've just downloaded it and it looks translated to me:
![image](https://user-images.githubusercontent.com/481846/163360635-79068e9e-5a34-432e-b10c-f1fdec963db2.png)

Please try the 'portable' package (no installer, just unzip and run from unzipped folder) to make sure you're running correct files.

**Comment by marcobra** [comment]
I've tried installer .run and tar.gz same result on linux ubuntu 20.04 i also clean the sals config in my homedir, i will try in windows in a while
![immagine](https://user-images.githubusercontent.com/1607387/163383845-b7719b34-9fda-4527-a261-4f73c26a3511.png)

**Reply by pawelsalawa** [reply]
Can you confirm that the language setting (the Italian) is correctly selected after you restarted the application and while it is still showing untranslated interface?

**Reply by marcobra** [reply]
yes

**Reply by marcobra** [reply]
Just testing in Windows right now and the translation works on same SqliteStudio build...
I will test next builds on Linux...

**Reply by pawelsalawa** [reply]
I will have to digg into it, but I'm not sure when I will have time to set up VM with Ubuntu locally.

**Reply by pawelsalawa** [reply]
Created #4433 to track it.

**Reply by marcobra** [reply]
i can help you to find the issue ... simply tell me the main steps and i will do test

**Reply by pawelsalawa** [reply]
I will probably have to install QtCreator, build it locally and debug...

**Reply by pawelsalawa** [reply]
Or perhaps - how about you record (using for example https://gifcap.dev/ ) how you download the package, decompress it and run it, then how you choose translation, etc.
Maybe I will spot something.

I know it's a lot to ask, because you would need to share the recording of your screen. If it would help, we could agree on sending the recording on email (support@sqlitestudio.pl), not to share it here.

**Reply by marcobra** [reply]
thanks i will do screen recording and i send to you...

**Comment by pawelsalawa** [comment]
This is fixed now and will be available in a weekly build (executed during the Sunday-Monday night).
